src/main/java/gui/Controller.java

```diff
@@ -114,17 +114,25 @@ public void createVideo(final File file) {
      * Call updateImageView method every period of time to retrieve a new frame
      * @param imageView panel that shows the frame
      */
-    public void grabTimeFrame(final ImageView imageView) {
+    void grabTimeFrame(final ImageView imageView) {
         if (!cameraActive) {
             ScheduledExecutorService timer;
             final int period = 1;
-            Runnable frameGrabber = () -> updateImageView(imageView);
+            Runnable frameGrabber = () -> updateViews(imageView);
             timer = Executors.newSingleThreadScheduledExecutor();
             timer.scheduleAtFixedRate(
                 frameGrabber, 0, period, TimeUnit.MILLISECONDS);
         }
     }
 
+    /**
+     * updateViews.
+     * @param imageView the view for the video
+     */
+    void updateViews(final ImageView imageView) {
```

Comment:
Maybe change this into updateView (for now)? Otherwise it may be somewhat confusing